### ISSUE TITLE
Can i just compile without deploying?

### ISSUE THREAD

**Issue by schosterbarak** [winglang/wing-github-action#10]
One example I can think of is
1. Compile to cdktf
2. Scan with policy as code tool (checkov/OPA/KICKS/ other)
3. If scan step is passing then deploy with wing

**Comment by skorfmann** [comment]
Yes, good point - that's missing right now. There should probably be the following actions:

- compile
- test
- destroy

Haven't tried it yet, but potentially the [validate](https://www.winglang.io/docs/tools/compiler-plugins#validate-hook) hook could be used for this as well without an explicit Github Action step.

**Comment by schosterbarak** [comment]
where are the compiled artifacts saved? whats the directory path?

**Comment by Chriscbr** [comment]
@schosterbarak The artifacts are saved to the `target` directory next to your code. For example, if you compile an app named "my-app" to the "tf-aws" target, your deployment artifacts will be inside `target/example.tfaws/`. I'll take note to add this to our docs.
